fulp_modules/main_features/bloodsuckers/code/mobs/clawmonster.dm

```diff
@@ -0,0 +1,35 @@
+/mob/living/simple_animal/hostile/retaliate/tzimisce_clawmonster
+	name = "clawed monster"
+	desc = "It's a very disturbing abomination. Is it looking at you?"
+	icon = 'icons/mob/animal.dmi' // Placeholder until actual sprite
+	icon_state = "tarantula"
+	icon_living = "tarantula"
+	icon_dead = "tarantula_dead"
+	maxHealth = 100
+	health = 100
+	melee_damage_lower = 20
+	melee_damage_upper = 25
+	melee_damage_type = BURN
+	attack_verb_continuous = list("claws", "slashes")
+	attack_verb_simple = list("claw", "slash")
+	speed = 0.5
+	turns_per_move = 5
+	see_in_dark = 4
+	attack_vis_effect = ATTACK_EFFECT_CLAW
+	initial_language_holder = /datum/language_holder/vampiric_mob
+	damage_coeff = list(BRUTE = 0.3, BURN = 0.9, TOX = 0, CLONE = 0, STAMINA = 0, OXY = 1)
+	atmos_requirements = list("min_oxy" = 0, "max_oxy" = 0, "min_tox" = 0, "max_tox" = 0, "min_co2" = 0, "max_co2" = 0, "min_n2" = 0, "max_n2" = 0)
+
+/mob/living/simple_animal/hostile/retaliate/tzimisce_clawmonster/examine(mob/user)
```

Comment:
you do this sort of stuff for each one, why not make a generic tzimisce subtype for all the shared stuff?